src/components/Pricing/Test/PurchasedWith.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { SectionHeader, section } from './Sections'
+
+const purchasedWith = [
+    {
+        name: 'Digital Ocean',
+        description: 'Cloud infrastructure for developers',
+        logo: 'https://res.cloudinary.com/dmukukwp6/image/upload/DO_Logo_icon_blue_0ade9109b2.svg',
+    },
+    {
+        name: 'Vercel',
+        description: 'Build and deploy web experiences',
+        logo: 'https://res.cloudinary.com/dmukukwp6/image/upload/vercel_icon_dark_1023f19025.svg',
+    },
+    {
+        name: 'Ashby',
+        description: 'All-in-one recruiting software for ambitious teams.',
+        logo: 'https://res.cloudinary.com/dmukukwp6/image/upload/ashby_e4a768e087.svg',
+    },
+    {
+        name: 'Algolia',
+        description: 'AI-powered search and discovery',
+        logo: 'https://res.cloudinary.com/dmukukwp6/image/upload/Algolia_mark_blue_f25812a7a2.svg',
+    },
+]
+
+export default function PurchasedWith() {
+    return (
+        <section className={section}>
+            <SectionHeader>
+                <h3 className="mb-2">Frequently purchased with...</h3>
```

Comment:
I think we need an explainer here. "These are some products that pair well with posthog to help you find PMF and maybe even get to an IPO."